### DISCUSSION TITLE
ETA on stable OpenTelemetry.Extensions.Hosting?

### DISCUSSION THREAD

**Discussion by mmisztal1980** [open-telemetry/opentelemetry-dotnet#3383] [Q&A]
I've just checked and it appears that the package in question is still available as a RC package. The milestone appears not to have been updated in a while. 

Does anyone have a rough idea when will the package will be released?

**Comment by cijothomas** [comment]
No estimates yet. Something we are looking to work on. Will update the milestone, when we have something concrete.

cc : @utpilla
